example/src/main/java/org/apache/rocketmq/example/benchmark/Consumer.java

```diff
@@ -111,7 +113,7 @@ public void run() {
             }
         }, 10000, 10000);
 
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(group);
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(group, AclClient.getAclRPCHook(), new AllocateMessageQueueAveragely(), msgTraceEnable, null);
```

Comment:
You need to maintain compatibility and let users choose whether to use ACL instead of forcing users to use ACL.

Reply:
OK, I will fix it ASAP.

Reply:
@RongtongJin  I‘ve fixed it, pls review it~